ipfs-search/dweb-search-frontend
# Vite migration

#### This PR migrates from webpack/vue-cli to vite.
It also replaces the testing package jest with vitest. (Jest does not play nice with vite, they are of different schools of js modules)
It does not (yet) update vue, vuex or vuetify. 

**N.b.** that the sass errors are still there when running `npm run serve` because vuetify has not been updated yet. This causes the build to be a bit slow. Still you should observe a pleasant speed increase of the build.

#### Migration log:
https://www.notion.so/ipfs-search/Upgrading-to-Vue3-part-1-migrating-to-Vite-cdc9fe3e553442199d53a4767cfcb4a9

#### To test this:
after `npm i`,
1. try running the code. See if all the functions of the frontend work properly. See if there are no bugs or console errors. (that were not there on the master branch)
2. Run all the scripts. See that they don't produce errors.
